rfcs/0021-glossary-terms.md

```diff
@@ -101,6 +92,41 @@ The terms proposed are below:
 - Trustless nodes - again, some lifting required.
 - Light node / light client - has a negative connotation of not being able to validate chain state.
 
+### User Transaction
+**Concept:** A transaction issued by a user - currently a payment or a delegation change.
+
+**Rationale:** This term clearly describes the concept, and leaves room for further types of user issued transactions.
+
+**Downsides:** This is a subset of "Transactions", so it is a bit redundant.
+
+**Usage:** There are three types of transactions in the Coda network currently - user transactions, fee transfers, and coinbases. User transactions allow users to manage accounts and send money.
+
+**Alternatives:**
+- User command - this term is confusing, especially in  docs, as it can be conflated with a CLI command that a user issues.
+
+### Protocol Transaction
+**Concept:** A transaction issued by the protocol that does not have a "from" field - currently a fee transfer or coinbase (both are structurally identical).
```

Comment:
This may be too limiting -- it happens to be the case that right now the protocol transactions don't have a from-field, but I think that may be accidental. I still like the name "protocol transaction"

Reply:
@bkase -- can I just remove that phrase, and would that be enough to land this?

Reply:
yup